homeassistant/helpers/llm.py

```diff
@@ -94,6 +94,13 @@ def async_register_api(hass: HomeAssistant, api: API) -> None:
 
     apis[api.id] = api
 
+    @callback
+    def unregister() -> None:
+        """Unregister the API."""
+        apis.pop(api.id)
+
+    return unregister
```

Comment:
Tests for this change is missing.

This change could be a separate PR.

Reply:
Good point, done #135162